ansible/roles_ocp_workloads/ocp4_workload_amq_streams/templates/installplan.j2

```diff
@@ -0,0 +1,7 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
You don't need that file (it's a mistake that it's still in the serverless workload)